src/wandbot/ingestion/vectorstores.py

```diff
@@ -55,45 +51,47 @@ def load(
         wandb.Error: An error occurred during the loading process.
     """
     config: VectorStoreConfig = VectorStoreConfig()
-    run: wandb.Run = wandb.init(
+    run: wandb.wandb_sdk.wandb_run.Run = wandb.init(
         project=project, entity=entity, job_type="create_vectorstore"
     )
     artifact: wandb.Artifact = run.use_artifact(
         source_artifact_path, type="dataset"
     )
     artifact_dir: str = artifact.download()
-    storage_context = load_storage_context(config.embedding_dim)
-    service_context = load_service_context(
-        embeddings_cache=str(config.embeddings_cache),
-        llm="gpt-3.5-turbo-16k-0613",
-        temperature=config.temperature,
-        max_retries=config.max_retries,
+
+    # Todo: Change to LiteLLM Embeddings
```

Comment:
is this TODO for the next PR?

Reply:
this is taken off in the wandbot-eval branch.